solr/core/src/test/org/apache/solr/util/SolrPluginUtilsTest.java

```diff
@@ -297,6 +298,16 @@ public void testMinShouldMatchCalculator() {
     assertEquals(1, calcMSM(4, "\n 3 < \n25%\n "));
     assertEquals(1, calcMSM(5, "3<25%"));
 
+    /* malformed conditionals */
+    try {
+      calcMSM(1, "1<");
+      fail("Query '1<' should have thrown SolrException");
+    } catch (SolrException expected) {}
+    try {
+      calcMSM(1, "1<x");
+      fail("Query '1<x' should have thrown SolrException");
+    } catch (SolrException expected) {}
+
```

Comment:
I'd move these lines into it's own test method

Reply:
Done: testMinShouldMatchBadQueries().